pages/deduplicate/index.js

```diff
@@ -0,0 +1,56 @@
+const bodyParser = require('body-parser');
+const { page } = require('@asl/service/ui');
+
+module.exports = settings => {
+  const app = page({
+    ...settings,
+    root: __dirname
+  });
+
+  app.use(bodyParser.urlencoded({ extended: true }));
+
+  app.get('/', (req, res, next) => {
+    req.breadcrumb('profile.view');
```

Comment:
Do we want another breadcrumb "Merge profiles" or some such so that the profile breadcrumb is a link back to the profile page instead of just text?

Reply:
Hard no.